docs/ds20.md

```diff
@@ -0,0 +1,132 @@
+# fwupd BOS DS20 Specification
```

Comment:
Don't forget to add a reference in `libfwupdplugin` docs and `index.html` docs too to this

Reply:
Added, thanks. Good catch.